sdk/aqueduct/responses.py

```diff
@@ -4,11 +4,32 @@
 
 from aqueduct.artifact import Artifact
 from aqueduct.dag import Metadata
-from aqueduct.enums import ExecutionStatus
+from aqueduct.enums import ExecutionStatus, FailureType
 from aqueduct.operators import Operator
 from aqueduct.utils import human_readable_timestamp
 
 
+class Logs(BaseModel):
+    stdout: str = ""
+    stderr: str = ""
+
+    def is_empty(self) -> bool:
+        return self.stdout == "" and self.stderr == ""
+
+    def __str__(self) -> str:
+        return f"""stdout:
+{self.stdout}
+--------------------------
+stderr:
+{self.stderr}
+"""
```

Comment:
I've been using `textwrap.dedent(..)` to avoid this really weird indentation. See `FlowRun.describe()`.